schema.prisma

```diff
@@ -184,12 +184,15 @@ model Campaign {
 }
 
 model CampaignFile {
-  id            String   @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
-  filename      String   @db.VarChar(200)
-  campaignId    String   @map("campaign_id") @db.Uuid
-  campaign      Campaign @relation(fields: [campaignId], references: [id])
-  uploadedById  String   @map("uploaded_by_id") @db.Uuid
-  uploadedBy    Person   @relation(fields: [uploadedById], references: [id])
+  id           String           @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  filename     String           @db.VarChar(200)
+  role         CampaignFileRole
+  type         String           @db.VarChar(200)
```

Comment:
This type is still string, maybe we should use `CampaignFileType` enum instead.

If `CampaignFileType` will not be used we better delete it